app/templates/index.hbs

```diff
@@ -45,6 +45,16 @@
             <td class="au-u-visible-medium-up">
               <AuLink @route="formbuilder.edit" @model={{generatedForm.id}} @icon="pencil" @iconAlignment="left" @ariaHidden={{true}}>Bewerk</AuLink>
             </td>
+            <td>
```

Comment:
There is now an extra column without a corresponding `th`, which causes a visual bug with the column heading. Setting `colspan="3"` on line 38 should fix it.